requirements_test.txt

```diff
@@ -5,7 +5,7 @@
 # may need to use this version of ramses_rf...
 # -e ~/client/ramses_rf
 
-# for using the the library with HA
+# for using the library with HA
```

Comment:
You have three PRs here:
 - one to add translations 
 - another to fix a typo (the the)
 - removing a comment

Generally, try and keep PRs distinct & atomic.